test/TestGlobalFunctions.h

```diff
@@ -1,9 +1,18 @@
 #include <Eigen/Dense>
 #include "mfem.hpp"
+#include "maxwell/Types.h"
+#include "maxwell/Model.h"
+#include "maxwell/BilinearIntegrators.h"
 
 using namespace mfem;
 
-Eigen::MatrixXd convertMFEMDenseToEigen(DenseMatrix* mat);
+struct FluxCoefficient {
+	double alpha;
+	double beta;
+};
+
+Eigen::MatrixXd convertMFEMDenseToEigen(const DenseMatrix* mat);
 Eigen::MatrixXd buildMassMatrixEigen(std::unique_ptr<FiniteElementSpace>& fes);
 Eigen::MatrixXd buildInverseMassMatrixEigen(std::unique_ptr<FiniteElementSpace>& fes);
-Eigen::MatrixXd buildStiffnessMatrixEigen(std::unique_ptr<FiniteElementSpace>& fes);
\ No newline at end of file
+Eigen::MatrixXd buildStiffnessMatrixEigen(std::unique_ptr<FiniteElementSpace>& fes);
```

Comment:
const FiniteElementSpace* ??

Reply:
Same comment as before, BilinearForm doesn't accept const FiniteElementSpace* as input argument, it could be a simple FiniteElementSpace or it could be left as is.